src/cat.cpp

```diff
@@ -1,5 +1,6 @@
 /* basename.cpp, migrated from basename.c from coreutils
    cat -- concatenate files and print on the standard output.
+*/
 
  /*
```

Comment:
Probably best to make similar to other files and drop the extra set of comments.  Thanks for the catch :smile: 
```suggestion
/* cat -- concatenate files and print on the standard output.
```